### PR TITLE
VirtualScroller bugfix: truncation indication, ConsoleError interactivity

### PR DESCRIPTION
**Pull request by adamconroy**
### Intent

To address: https://github.com/rstudio/rstudio/issues/8940

I've had to rewrite the element manipulation to be compatible with raw Javascript. Unfortunately, when an element is added to the DOM through an external source, GWT loses the bindings are we aren't able to manipulate the elements through GWT variables anymore. I rewrote the functionality with plain DOM manipulation functions.

To address: https://github.com/rstudio/rstudio/issues/7918

I've added `[...]` to the top of the output to indicate that it has been truncated. I'm open to ideas for alternatives to put there but this is very similar to what Visual Studio does when outputting large collections. I don't think clearing the console cutting off everything (including the command prompt) like we do without the VirtualScroller is the right move as it kind of defeats the purpose of the feature entirely.

RexuPhv

### Automated Tests

There are no automated tests for this feature as there is already some coverage.

### QA Notes

Verify that ConsoleErrors with traceback function properly with VirtualScroller both on and off.

### Checklist

- [x ] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [x ] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [x ] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [x ] This PR passes all local unit tests

### CODE REVIEW

File: src/gwt/src/org/rstudio/studio/client/common/debugging/ui/ConsoleError.ui.xml [hunk 1]
```diff
@@ -74,8 +74,8 @@
                   ui:field="showTracebackImage"
                   width="9"
                   height="9"
-                  styleName="{style.errorCommandImage}"/>
-           <g:Anchor styleName="{style.errorCommandText}" ui:field="showTracebackText">
+                  styleName="{style.errorCommandImage} show_traceback_image"/>
```
Comment (kevinushey) [inline src/gwt/src/org/rstudio/studio/client/common/debugging/ui/ConsoleError.ui.xml:77]: Can you clarify why we need these extra class names here? Are we unable to query for the class name generated by GWT in the associated `style` object?

**Review by kevinushey — APPROVED** [review]
LGTM; thank you for slogging through this!

My main comment is whether the separate hard-coded class names are necessary, or if we could still query / use the style names generated by GWT for each UI element.

File: src/gwt/src/org/rstudio/studio/client/common/debugging/ui/ConsoleError.ui.xml (hunk 1 above)
Comment (adamconroy) [reply src/gwt/src/org/rstudio/studio/client/common/debugging/ui/ConsoleError.ui.xml:77]: I just double checked and unfortunately GWT is assigning the same class to the bindings, both tex telements "Show Traceback" and "Rerun" have the class `org-rstudio-studio-client-common-debugging-ui-ConsoleError_ConsoleErrorUiBinderImpl_GenCss_style-errorCommandText`.

**Review by gtritchie — APPROVED** [review]
LGTM! Sorry for delay, I looked at it yesterday and forgot to actually approve it.

**Pull request merged**